src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadTaxationData.php

```diff
@@ -91,7 +92,7 @@ protected function createTaxCategory($name, $description)
      *
      * @return TaxRateInterface
      */
-    protected function createTaxRate($name, $zoneName, $amount, $includedInPrice = false, $calculator = 'default')
+    protected function createTaxRate($name, $code, $zoneName, $amount, $includedInPrice = false, $calculator = 'default')
```

Comment:
The `code` parameter should be the first, as it is in the `createTaxCategory` method.